tests/test_tsfresh.py

```diff
@@ -0,0 +1,186 @@
+import polars as pl
+import pytest
+from polars.testing import assert_frame_equal
+
+from functime.feature_extraction.tsfresh import (
+    autocorrelation,
+    count_above,
+    count_above_mean,
+    count_below,
+    count_below_mean,
+    has_duplicate,
+    has_duplicate_max,
+    has_duplicate_min,
+)
+
+
+@pytest.fixture
+def rental_car_dataset():
+    return pl.DataFrame(
```

Comment:
Try testing both `DataFrame` and `LazyFrame`. You can do this nicely using `pytest.fixture(params=...)`. Check out the docs: https://docs.pytest.org/en/7.3.x/how-to/fixtures.html#fixture-parametrize. Here is what it'll look like:

```python
@pytest.fixture(params=[pl.DataFrame, pl.LazyFrame])
def rental_car_dataset(request):
    df_cls = request.param
    return df_cls(...)
```